deploy/ubuntu-config/proxy.ini

```diff
@@ -1,6 +1,12 @@
 [privacyidea]
 # URL of your privacyIDEA installation
 instance = http://192.0.2.2
+# If you use HTTPS, you can manually pass a certificate (PEM format)
+# If this option is omitted, the presented certificate is validated against the OS certificate
+# store.
+#certificate = /etc/privacyidea/server.pem
+# You can also disable hostname validation completely, which is not recommended.
```

Comment:
It is not quite clear, that ``verify`` needs to be set to ``True`` in order for ``certificate`` to take effect.
Maybe you can add something like:

* Default of ``verify`` is True
* If you want the manually passed certificate to be verified, you need to set ``verify=True``.